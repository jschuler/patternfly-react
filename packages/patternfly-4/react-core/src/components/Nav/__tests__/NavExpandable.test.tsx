// Auto-generated do not edit

import * as React from 'react';
import { mount } from 'enzyme';
import { NavExpandable } from '../NavExpandable';

describe('NavExpandable test', () => {
  it('NavExpandable should match snapshot', () => {
    const view = mount(
      <NavExpandable
        title={"'defaultString'"}
				srText={"''"}
				isExpanded={false}
				children={''}
				className={"''"}
				groupId={null}
				isActive={false}
				id={"''"}
				onExpand={undefined}
      />);
    expect(view).toMatchSnapshot();
  });
});
