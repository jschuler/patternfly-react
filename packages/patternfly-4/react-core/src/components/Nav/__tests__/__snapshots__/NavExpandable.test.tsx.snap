// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavExpandable test NavExpandable should match snapshot 1`] = `
<NavExpandable
  className="''"
  groupId={null}
  id="''"
  isActive={false}
  isExpanded={false}
  srText="''"
  title="'defaultString'"
>
  <li
    className="pf-c-nav__item ''"
    onClick={[Function]}
  >
    <a
      aria-expanded={false}
      className="pf-c-nav__link"
      data-component="pf-nav-expandable"
      href="#"
      id={null}
      onClick={[Function]}
      onMouseDown={[Function]}
    >
      'defaultString'
      <span
        className="pf-c-nav__toggle"
      >
        <AngleRightIcon
          aria-hidden="true"
          color="currentColor"
          noVerticalAlign={false}
          size="sm"
          title={null}
        >
          <svg
            aria-hidden="true"
            aria-labelledby={null}
            fill="currentColor"
            height="1em"
            role="img"
            style={
              Object {
                "verticalAlign": "-0.125em",
              }
            }
            viewBox="0 0 256 512"
            width="1em"
          >
            <path
              d="M224.3 273l-136 136c-9.4 9.4-24.6 9.4-33.9 0l-22.6-22.6c-9.4-9.4-9.4-24.6 0-33.9l96.4-96.4-96.4-96.4c-9.4-9.4-9.4-24.6 0-33.9L54.3 103c9.4-9.4 24.6-9.4 33.9 0l136 136c9.5 9.4 9.5 24.6.1 34z"
              transform=""
            />
          </svg>
        </AngleRightIcon>
      </span>
    </a>
    <section
      aria-labelledby="''"
      className="pf-c-nav__subnav"
      hidden={true}
    >
      <h2
        className="pf-u-screen-reader"
        id="''"
      >
        ''
      </h2>
      <ul
        className="pf-c-nav__simple-list"
      />
    </section>
  </li>
</NavExpandable>
`;
